### ISSUE TITLE
bug: module 'deepxde.nn.pytorch' has no attribute 'MfNN'

### ISSUE THREAD

**Issue by lilux618** [lululxvi/deepxde#1105]
When run the make script in examples in deepxde  with pytorch backend, there is an error as follow: 
<img width="603" alt="image" src="https://user-images.githubusercontent.com/60692776/211452767-01d2dace-5c09-4224-8c12-318c9921e469.png">

**Comment by tsarikahin** [comment]
Yes,  `mfNN` does not exist for Torch. You can set your backend as `tensorflow_compat_1`. It should work
